checkly/resource_private_locations_test.go

```diff
@@ -25,7 +25,7 @@ func TestAccPrivateLocationSuccess(t *testing.T) {
 	config := `resource "checkly_private_location" "test" {
 		name     = "New Private Location"
 		slug_name   = "new-private-location"
-		icon       	= "location"
+		icon       	= "bell-fill"
```

Comment:
Hi RealOrangeOne! Thanks for your contribution. One question out of curiosity, why are you setting a different icon here? just for test purposes? Thanks again and we will review this asap.

Reply:
I changed this here so it shows that the provided value is flowing through correctly, rather than just using the default. The new test I added makes sure the default value is used if `icon` is omitted.